macOS/LocalPackages/FeatureFlags/Sources/FeatureFlags/FeatureFlag.swift

```diff
@@ -53,6 +53,8 @@ public enum FeatureFlag: String, CaseIterable {
     /// https://app.asana.com/0/1201048563534612/1208850443048685/f
     case historyView
 
+    case autoUpdateInDEBUG
```

Comment:
Nice, that's a good workaround for facilitating easier debugging without having to modify the codebase every time.